app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivityScreen.kt

```diff
@@ -0,0 +1,184 @@
+package com.github.se.travelpouch.ui.dashboard
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.ArrowBack
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.icons.filled.DateRange
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material3.BottomAppBar
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.github.se.travelpouch.model.Location
+import com.github.se.travelpouch.model.activity.Activity
+import com.github.se.travelpouch.model.activity.ActivityModelView
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import com.google.android.material.bottomnavigation.BottomNavigationItemView
+import com.google.android.material.bottomnavigation.BottomNavigationMenuView
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.navigation.NavigationBarItemView
+import com.google.firebase.Timestamp
+import java.util.Calendar
+import java.util.GregorianCalendar
+
+data class BottomNavigationItem(
+    val title: String,
+    val icon: ImageVector
+)
+
+@Preview
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TravelActivitiesScreen(
+    navigationActions: NavigationActions? = null,
```

Comment:
Why did you add null as a default value for navigationActions?

Reply:
We still need to discuss the navigation issue (with route and screen). So I wanted to have a functional view to work with. But, at the moment when the navigation issue is solved, I will make this parameter non-nullable.

Reply:
ok I see